### ISSUE TITLE
Dino fails in chrome & firefox -- constant pane resizing, 100% cpu spin, unresponsive UI

### ISSUE THREAD

**Issue by planetf1** [odpi/egeria#4123]
* Launch coco pharm lab environment (I tried both docker-compose & k8s)
* Run through config/start of servers
* Launch dino UI ( /coco/login), login, go to Dino
* Observe that the window 'shakes' as the main content pane constantly resizes. The browser CPU spins a core, and UI is unresponsive. 

This occurs under firefox & chrome (current beta versions) but not under safari (Big sur)

I wonder if the recent layout change may have caused this -- 

Also affects type explorer, repository explorer which are completely unresponsive

Glossary author layout is working ok

Version: The latest master (HEAD @ 8d2589615334548af92cd0e68426480bc4171500 )

**Comment by grahamwallis** [comment]
I have tested Rex, Tex and Dino in Chrome and they seem to work exactly as they do in Safari. Everything lays out correctly and there is no resizing or oscillation. There's also no heavy CPU load.
I have not tested with FF.

I did not use the coco-pharma environment but could try that if you think it would make a difference.

**Comment by planetf1** [comment]
I have tried again with chrome & still got the issue.
Wondering if it was my environment/plugins I then tried with open source 'chromium' (so no google stuff, unconfigured so no plugins) - same issue.
Will check on another machine and update...

**Comment by grahamwallis** [comment]
@planetf1 - Thanks Nigel. If it helps I am running Chrome Version 87.0.4280.67 (Official Build) (x86_64) in a completely vanilla install.

**Comment by planetf1** [comment]
Also get this on windows chrome & edge. Current chrome is 87.0.4280.67 also

I wonder if it could be the container environment itself -- ie exact version of dependencies/node in use -- though that should be consistent. 

Will send you URL to check

**Comment by planetf1** [comment]
I made the window pretty small, but at 4K native resolution, with some Mac scaling (1205x520) and it rendered ok.

At 1247 (wide) x 1028 it fails. Just two samples

Definitely seems layout related

**Comment by planetf1** [comment]
Able to reproduce on safari, but is dependent on resolution.

In Safari go to Develop->Enter Responsive Design Mode

Select 1920x1080 resolution & go to https://<site>:8091/coco/login / garygeeke / dino -> Bounces/fails
Select 1366 x 768 resolution & again go to that page/garygeeke/dino

**Comment by planetf1** [comment]
This problem does not seem to occur with release 2.4 code, ie a regression in 2.5
Comparing the docker images ie using commands like
```
docker run odpi/egeria-presentation-server:2.4-SNAPSHOT
docker run odpi/egeria-presentation-server:2.4-SNAPSHOT
docker ps
docker exec -it container-id /bin/sh
```
2.4 package.json -> https://gist.github.com/f23abe639b0d93d1716ec596bc4357b9
2.5 package.json -> https://gist.github.com/387ca2dd56df7370c8ca74687b8ba2ca

**Comment by grahamwallis** [comment]
Fixed by #4155
